ansible/roles/master/meta/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 dependencies:
     - { role: common }
+    - { role: docker }
```

Comment:
That is required because of the kubelet running on the master. Can i make this conditional ?